vector-search-simulator/vector-test/vectors-datasets-tests-v2.yaml

```diff
@@ -0,0 +1,51 @@
+- name: read_only
+  duration: 30m
+  repetitions: 1
+  clients: 1
+  members: 1
+  driver: hazelcast-enterprise5
+  version: maven=5.5.0-SNAPSHOT
+  client_args: >
+    -Xms30g
+    -Xmx30g
+    --add-modules java.se 
+    --add-exports java.base/jdk.internal.ref=ALL-UNNAMED 
+    --add-opens java.base/java.lang=ALL-UNNAMED 
+    --add-opens java.base/sun.nio.ch=ALL-UNNAMED 
+    --add-opens java.management/sun.management=ALL-UNNAMED 
+    --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED
+  member_args: >
+    -Xms30g
```

Comment:
I assume that in your report you used this config, not the other one with 3GB of heap